### PR TITLE
Minor changes to logging and add variable to display ascii logo

### PR DESCRIPTION
**Pull request by anthonychu**
Some small changes while investigating options for #2222. @pragnagopa let me know what you think.

* Remove ASP.NET Core startup logs if not verbose
* Change formatting to make the beginning of the command stand out a bit more
* Change timestamps to gray to make them less prominent
* Add `FUNCTIONS_CORE_TOOLS_DISPLAY_LOGO` to force ASCII art if set to `1`

image

### CODE REVIEW

File: src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs [hunk 1]
```diff
@@ -253,10 +253,18 @@ private void UpdateEnvironmentVariables(IDictionary<string, string> secrets)
...
-            if (VerboseLogging.HasValue && VerboseLogging.Value)
+            var displayLogo = Environment.GetEnvironmentVariable(Constants.DisplayLogoVariable) == "1";
+            var isVerbose = VerboseLogging.HasValue && VerboseLogging.Value;
+            if (isVerbose || displayLogo)
             {
                 Utilities.PrintLogo();
             }
+
+            if (!isVerbose)
+            {
```
Comment (pragnagopa) [inline src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs:264]: good to add E2E tests.  If its ok, I can add tests and push to this branch?

**Review by pragnagopa — COMMENTED** [review]
changes in colors look good! Added minor comments.

File: src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs (hunk 1 above)
Comment (anthonychu) [reply src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs:265]: Yes. Thanks!

**Review by pragnagopa — APPROVED** [review]

**Review by ankitkumarr — APPROVED** [review]

**Pull request merged**